Updated the k8s-ei-operator documentation (#1028)
wso2/docs-ei: en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md

## Prerequisites
The k8s-ei-operator is built with operator-sdk v0.7.0 and supported in the following environment.

-   [Kubernetes](https://kubernetes.io/docs/setup/) cluster and client v1.11+   
-   [Docker](https://docs.docker.com/)

## Install EI Operator

> Please do the following steps to intall the EI Kubernetes operator in your Kubernetes environment.

Reviewer: ```suggestion
Please do the following steps to install the EI Kubernetes operator in your Kubernetes environment.
```

Author: Done

1.  Clone the **k8s-ei-operator** GitHub repository.

    ```bash
    git clone https://github.com/wso2/k8s-ei-operator.git
    ```
    
2.  Change directory to k8s-ei-operator.

    ```bash
    cd k8s-ei-operator
    ```
    
3.  Setup Service Account.

    ```bash
    kubectl create -f deploy/service_account.yaml
    ```
    
5.  Setup RBAC.

    ```bash
    kubectl create -f deploy/role.yaml
    kubectl create -f deploy/role_binding.yaml
    ```
    
6.  Deploy CustomResourceDefinition into the Kubernetes cluster to understand custom resource type.

    ```bash
    kubectl create -f deploy/crds/integration_v1alpha1_integration_crd.yaml
    ```
    
7.  Deploy the k8s-ei-operator.

    ```bash
    kubectl create -f deploy/operator.yaml
    ```
    
8. Apply configuration for the ingress controller

    ```bash
    kubectl apply -f deploy/config_map.yaml
    ```

You can verify the installation by making sure the following deployment are running in your Kubernetes cluster.

```bash
$ kubectl get deployment

NAME               READY   UP-TO-DATE   AVAILABLE   AGE
k8s-ei-operator     1/1     1            1          1m
```

## Deploy and Run Integration Solutions with EI Operator

> EI operator supports to deploy and run your integration solutions in Kubernetes environemnt. 

Reviewer: ```suggestion
EI operator supports deployment and running your integration solutions in a Kubernetes environment. 
```

Author: Done

> Here we have demonstrate an example of **HelloWorld** service which response as `{"Hello":"World"}` to the user request. This scenario can be created using `integration_cr.yaml` format as given below.

Reviewer: ```suggestion
Here we have demonstrated an example of **HelloWorld** service which responds `{"Hello": "World"}` to the user request. This scenario can be created using `integration_cr.yaml` format as given below.
```

Author: Done
